index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Chain of Positivity</title>
+</head>
+	<body>
+		<script src='https://api.tiles.mapbox.com/mapbox.js/v2.2.4/mapbox.js'></script>
+		<link href='hsttps://api.tiles.mapbox.com/mapbox.js/v2.2.4/mapbox.css' rel='stylesheet' />
+		<link rel="stylesheet" type="text/css" href="style.css">
+		<link rel="stylesheet" type="text/css" href="style.css">
+		<script src="scripts.js"></script>
+
+
+		<div id='map'>
+		Map
+		</div>
+		<iframe width='100%' height='500px' frameBorder='0' src='https://a.tiles.mapbox.com/v4/ciiyan.p2l0jh16/attribution,zoompan,zoomwheel,geocoder,share.html?access_token=pk.eyJ1IjoiY2lpeWFuIiwiYSI6Iks0djZqb0kifQ.KDk4q7pEJOORGrH-54g0jQ'></iframe>
+
+	</body>
+</html>
+
+<!-- //wide grey line route. 
+//we'll know what distance is, then for each team/org/ind that submits a chain,
+it captures name, group and links + photo
+//value with each link in a chain e.g meter. multiply links by distance. segmant per goal route
+//JSON has entries
+//photo, links, name of team
+//SEGMANTS
+//statehighway 1, coming from christchurch
+
+//distance chch to welly / 1000 -->
+
```

Comment:
I love the fact you documented this. Although, the readme is probably a better place for these types of comments. Easier to find later.